tests/layer_tests/tensorflow_tests/test_tf_BitwiseShift.py

```diff
@@ -61,15 +61,13 @@ def create_bitwise_net(self, x_shape, y_shape, is_y_const, input_type, op_type):
     @pytest.mark.parametrize('x_shape', [[4], [3, 4], [1, 2, 3, 4]])
     @pytest.mark.parametrize('y_shape', [[1], [4], [2, 3, 4]])
     @pytest.mark.parametrize('is_y_const', [True, False])
-    @pytest.mark.parametrize('input_type', [np.int8, np.int16, np.int32, np.int64,
-                                            np.uint8, np.uint16, np.uint32, np.uint64])
+    @pytest.mark.parametrize('input_type', [np.int8, np.int32, np.int64,
+                                            np.uint8, np.uint32])
```

Comment:
please revert this change. We should cover all possible type.

Reply:
The GPU plugin does not have uint64 data type implemented. Currently bitwise shift in GPU plugin does not support also in16 and uint16, but I am adding it.

Reply:
Please skip these cases properly and indicate a ticket number for adding these types for CPU:
```
if ie_device == 'GPU' and input_type in [...]:
    pytest.skip('...')
```

Reply:
Fixed.